packages/components/psammead-radio-schedule/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Radio Schedule Changelog
+
+| Version | Description |
+|---------|-------------|
+| 0.1.0-alpha.1 | [PR#2979](https://github.com/BBC-News/psammead/pull/2979) Initial creation of package. |
+| 0.1.0-alpha.2 | [PR#2938](https://github.com/BBC-News/psammead/pull/2938) Add radio program card component. |
```

Comment:
Thanks for pointing this out, I created `0.1.0-alpha.1` on a separate branch so that guys working on `radio-schedules` could have a common ground, I just pulled that branch into this one.